src/redux/books/bookSlice.js

```diff
@@ -1,34 +1,92 @@
-import { createSlice } from '@reduxjs/toolkit';
-import bookList from '../../bookList';
+/* eslint-disable no-param-reassign */
```

Comment:
- [ ] I noticed that you disabled **Eslint** . Please remove the **eslint disable** statement and try to fix all the errors you are getting. Because ` Eslint` helps developers to write clean JavaScript codes and adapt best practices when writing the code.  :+1: